SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -535,4 +538,4 @@ BusScopeSynth {
 		buffer = nil;
 		synth = nil;
 	}
-}
+}
```

Comment:
Please try not to modify the EOF newline, [as described here](https://github.com/supercollider/supercollider/wiki/Style-Guidelines:-SuperCollider#rule-use-exactly-one-newline-at-the-end-of-a-file).

Reply:
Done! Sorry, I knew that, but I did not know how to fix it. Anyway, I solved the problem by replacing the previous file. Then I added the `bounds_` method again.